pkg/entities/entities_api.go

```diff
@@ -2079,36 +2079,64 @@ const getEntitySearchByQuery = `query(
 			permalink
 			reporting
 			type
+			tags {
+				key
+				values
+			}
```

Comment:
we might not need this, can you please remove it (we can add it later when necessary)